.github/workflows/ci-cd.yaml

```
@@ -1,4 +1,4 @@
-name: Continous Integration and Demo Deployment
+name: Continuos Integration and Demo Deployment
```

funnily enough, it's still a typo 😬 
```suggestion
name: Continuous Integration and Demo Deployment
```